system/jlib/jdebug.cpp

```diff
@@ -947,20 +952,20 @@ memsize_t getMapInfo(const char *type)
         bool skipline = true;
         if ( !strcmp(type, "heap") ) // 'general' heap includes anon mmapped + sbrk
         {
-            if ( strstr(ln, "[heap]") || (!strstr(ln, " /") && !strstr(ln, " [")) )
+            if ( strstr(ln, "[heap") || (!strstr(ln, " /") && !strstr(ln, " [")) )
```

Comment:
It should be "[stack" and "[heap" as there can be more than one of these and they have format "[stack:<tid>]"
I'll make sure its correct in the other PR also.